src/Router.js

```diff
@@ -79,6 +79,7 @@ export default class TipsiRouter {
     const location = {
       pathname: compile(path)(paramsOrOptions),
       search: query && stringify(query, { addQueryPrefix: true }),
+      state: paramsOrOptions,
```

Comment:
Before you change it was https://github.com/tipsi/tipsi-router/blob/aff5589412b4d9632a2ecd085377c923f585d299/src/Router.js#L67
To be compatible with `ex-navigation`